src/AppInstallerCommonCore/Public/winget/ManifestValidation.h

```diff
@@ -249,8 +251,10 @@ namespace AppInstaller::Manifest
         bool m_warningOnly;
     };
 
+    static const std::unordered_set<std::filesystem::path> allowedPortableFiletypes = {".exe"};

```

Comment:
```suggestion
    static const std::unordered_set<std::filesystem::path> allowedPortableFiletypes = { L".exe" };

```
nit: Windows uses wide strings